ui/components/multichain/asset-picker-amount/asset-picker/asset-picker.tsx

```diff
@@ -107,7 +103,7 @@ export function AssetPicker({
   customTokenListGenerator,
   isTokenListLoading = false,
   isMultiselectEnabled = false,
-  autoFocus = true,
+  dataTestId = 'asset-picker-button',
```

Comment:
Nit: This is an unconventional way to use `dataTestId`. Typically this prop represents the test ID for the component, but here it's used for a sub-component (the asset picker button specifically).

Maybe it could be called `buttonDataTestId` instead? Or we could take a `dataTestId` here, but use `${dataTestId}-button` for the button?